docs/source/tutorials/base/how_to_train/detection.rst

```diff
@@ -0,0 +1,385 @@
+It allows to efficiently run it on Intel hardware, especially on CPU. Also, the resulting IR model is required to run POT optimization in the section below. IR model contains of 2 files: ``openvino.xml`` for weights and ``openvino.bin`` for architecture.
+
+2. That's how we can export the trained model ``outputs/weights.pth`` from the previous section and save the exported model to the ``outputs/openvino/`` folder.
+
+.. code-block::
+
+  (detection) ...$ otx export otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                              --load-weights outputs/weights.pth
+                              --save-model-to outputs/openvino/
+
+  ...
+
+  2023-01-10 06:23:41,621 | INFO : run task done.
+  2023-01-10 06:23:41,630 | INFO : Exporting completed
+
+
+3. We can check the accuracy of the IR model and the consistency between the exported model and the PyTorch model, using ``otx eval`` and passing IR model path to ``--load-weights`` parameter.
+
+.. code-block::
+
+  (detection) ...$ otx eval otx/algorithms/detection/configs/detection/mobilenetv2_atss/template.yaml
+                            --test-ann-files data/wgisd/coco_annotations/test_bbox_instances.json 
+                            --test-data-roots data/wgisd/data 
+                            --load-weights outputs/openvino/openvino.xml
+                            --save-performance outputs/openvino/performance.json
+  
+  ...
+  2023-01-10 06:24:50,382 | INFO : Start OpenVINO inference
+  2023-01-10 06:24:54,943 | INFO : OpenVINO inference completed
+  2023-01-10 06:24:54,944 | INFO : Start OpenVINO metric evaluation
+  2023-01-10 06:24:55,117 | INFO : OpenVINO metric evaluation completed
+  Performance(score: 0.5487693710118504, dashboard: (1 metric groups))
+
+
+*************
+Optimization
+*************
+
+1. We can further optimize the model with ``otx optimize``. It uses NNCF or POT depending on the model format.
+
+``NNCF`` optimization is used for trained snapshots in a framework-specific format such as checkpoint (pth) file from Pytorch. It starts accuracy-aware quantization based on the obtained weights from the training stage. Generally, we will see the same output as during training.
+
+``POT`` optimization is used for models exported in the OpenVINO™ IR format. It decreases floating-point precision to integer precision of the exported model by performing the post-training optimization.
+
+The function results with the following files, which could be used to run :doc:`otx demo <../demo>`:
+
+- ``confidence_threshold``
+- ``config.json``
+- ``label_schema.json``
+- ``openvino.bin``
+- ``openvino.xml``
+
+To learn more about optimization, refer to `NNCF repository <https://github.com/openvinotoolkit/nncf>`_.
+
+2. Command example for optimizing a PyTorch model (`.pth`) with OpenVINO NNCF. We can also simplify the commanline by adding ``--data data.yaml`` parameter instead of specifying training and validation paths there.
```

Comment:
```suggestion
2. Command example for optimizing a PyTorch model (`.pth`) with OpenVINO™ NNCF. We can also simplify the command line by adding ``--data data.yaml`` parameter instead of specifying training and validation paths there.
```